.github/workflows/oclif-release.yml

```diff
@@ -0,0 +1,64 @@
+name: Oclif Release
+on:
+  workflow_dispatch:
+    inputs:
+      home-brew-branch:
+        description: 'Run workflow in home brew branch'
+        default: main
+      tag-name:
+        description: 'Tag released recently'
+        required: true
+      formula:
+          description: 'Formula to release in homebrew'
+          default: twilio
+      pre-release:
+        description: 'Homebrew release type'
+        default: 'false'
+jobs:
+  oclif-release:
+    name: Publish for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    outputs:
+      sha256: ${{ steps.sha256.outputs.sha256 }}
+    strategy:
```

Comment:
Is this required? Don't see it mentioned in homebrew-release job. If yes, do we also need to have a non-fail-fast strategy at the jobs level too? Please help me understand what's the benefit of having it as false ?

Reply:
Currently as part os this PR, I have added only `npx oclif-dev pack`, but we need to add other commands specific to each OS like `npx oclif-dev pack:deb`, `npx oclif-dev pack:win` and `npx oclif-dev pack:macos`. Added `fail-fast: false` because of GitHub should not cancel other in-progress jobs if any matrix job fails.

Reply:
Okay, makes sense. Also, while doing homebrew-release would we need the same divisons there? For different packages? Lets say pack:win fails, we'd still be releasing other OS packages?

Reply:
Currently for homebrew release we just need the assets of  `npx oclif-dev pack` and `npx oclif-dev publish` will push assets to s3, from s3 Homebrew will use the assets. Other commands (`npx oclif-dev pack:deb`, `npx oclif-dev pack:win` and `npx oclif-dev`) we will use them to publish to github release  as part of [this story](https://issues.corp.twilio.com/browse/DII-70) not required for homebrew release.